cmd/crowdsec/main.go

```diff
@@ -183,6 +184,7 @@ func (f *Flags) Parse() {
 	flag.BoolVar(&f.TraceLevel, "trace", false, "VERY verbose")
 	flag.BoolVar(&f.DebugLevel, "debug", false, "print debug-level on stdout")
 	flag.BoolVar(&f.InfoLevel, "info", false, "print info-level on stdout")
+	flag.BoolVar(&f.DebugLevel, "warning", false, "print warning-level on stdout")
```

Comment:
you should pass `WarnLevel` var to store the value.

Reply:
you mean I only had to do one thing, and I botched it? I can't stop laughing